qcodes/instrument_drivers/tektronix/Keithley_6500.py

```diff
@@ -0,0 +1,185 @@
+from functools import partial
+from typing import Union
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import Bool, Enum, Ints, MultiType, Numbers
+
+
+def _parse_output_string(string_value: str):
+    """ Parses and cleans string output of the multimeter. Removes the surrounding
+        whitespace, newline characters and quotes from the parsed data. Some results
+        are converted for readablitity (e.g. mov changes to moving).
+
+    Args:
+        string_value: The data returned from the multimeter reading commands.
+
+    Returns:
+        The cleaned-up output of the multimeter.
+    """
+    s = string_value.strip().lower()
+    if (s[0] == s[-1]) and s.startswith(("'", '"')):
+        s = s[1:-1]
+
+    conversions = {'mov': 'moving', 'rep': 'repeat'}
+    if s in conversions.keys():
+        s = conversions[s]
+    return s
+
+
+def _parse_output_bool(numeric_value: Union[int, float]):
```

Comment:
There is no need for this function, right? `bool` itself will work just as good and will save 10 lines of code, and will make the rest of the driver more readable.